src/main/java/core/basesyntax/WorkWithBytes.java

```diff
@@ -1,7 +1,16 @@
 package core.basesyntax;
 
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+
 public class WorkWithBytes {
     public void writeBytesToFile(String fileName, byte[] data) {
-
+        File fileTo = new File(fileName);
+        try {
+            Files.write(fileTo.toPath(), data);
+        } catch (IOException e) {
+            throw new RuntimeException("Can not write to file");
```

Comment:
let's add the original exception as the cause

Reply:
added the exception and the filename to the exception object